src/css/fluidplayer.css

```diff
@@ -723,6 +722,62 @@ _:-ms-lang(x),
     user-select: none;
 }
 
+.fluid_video_wrapper.fluid_player_layout_default .fluid_controls_container .fluid_controls_right .fluid_control_duration  {
+    display: flex;
+    align-items: center;
+}
+
+.fluid_button_live_indicator {
```

Comment:
Can you add a `margin-bottom: 3px` to this. Atm it isn't aligned quite right. 

Before
![Screenshot 2024-12-02 at 14 58 34](https://github.com/user-attachments/assets/db00c129-b662-494e-843f-21d9039438af)

After
![Screenshot 2024-12-02 at 14 57 58](https://github.com/user-attachments/assets/5a705192-0d30-4da6-bffe-99b651cac66d)